kafka-client/src/test/java/dev/responsive/kafka/api/ResponsiveKafkaStreamsTest.java

```diff
@@ -154,8 +155,8 @@ public void shouldInvalidateBadConfigs() {
     builder.stream("foo").to("bar");
 
     // Then:
-    final StreamsException e = assertThrows(
-        StreamsException.class,
+    final ConfigException e = assertThrows(
```

Comment:
I intentionally removed an outer layer of wrapping ConfigException in a StreamsException since it didn't really add any additional info and just obfuscated the actual error